site/content/docs/5.3/components/orange-navbar.md

```diff
@@ -376,6 +376,87 @@ You can add a search input into your Global header.
 </header>
 ```
 
+### With theme selector
+
+You can add a theme selector control into your Global header.
+
+{{< callout info >}}
+Make sure to import the icons inside your project.
+
+Make sure our [theme javascript]({{< docsref "/customize/color-modes#javascript" >}}) is imported or mocked as well.
+
+Some small css needs to be added:
+```css
+#global-header-6.2 .dropdown-menu .active .bi {
```

Comment:
IMO developers shouldn't have to add some CSS manually.
The only thing they should do is add an ID and adapt the `color-modes.js` to use this ID.